models/news-model.js

```diff
@@ -53,19 +56,20 @@ exports.insertNewComment = (newComment, article_id) => {
     .query(checkingUserQuery, [username])
     .then((authorsArray) => {
       if (authorsArray.rows.length === 0) {
-        return Promise.reject();
+        return Promise.reject({ status: 404, msg: "Error: 404 not found" });
       }
-      return authorsArray.rows[0];
+      return Promise.all(authorsArray.rows);
     })
```

Comment:
This seems little unnecessary - authorsArray.rows is an array of authors, not pending promises, so you can just carry on without the need for a Promise.all()/.then block